src/scripts/ping.ts

```diff
@@ -67,22 +71,28 @@ export default (
   rootCommand.command(
     'ping',
     // eslint-disable-next-line quotes
-    "Pings the okta server to see if it's running and check user credentials",
+    "Pings the okta server to see if it's running and check user credentials along with organisation url",
     // eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types
     (yargs) => yargs.argv,
     async (args: {
       readonly clientId: string;
       readonly privateKey: string;
       readonly organisationUrl: string;
     }) => {
-      const client = oktaReadOnlyClient({ ...args });
-
-      const { clientId, organisationUrl } = args;
-      const result = await validateOktaServerAndCredentials(
-        client,
-        clientId,
-        organisationUrl
+      const { organisationUrl, clientId } = args;
+      const result = await parseUrlWrapper(organisationUrl, (url: string) =>
```

Comment:
**Issue:** `organisationUrl` is defined in one place (`src/index.ts`). It is a bit of a code smell to have it defined in one place but then parsed in all the commands.

Is there a way for you to parse the `organisationUrl` in one place?

Reply:
modified in commit: fef29bd8f6e2900782da9d97661727803150f786
This is a much better way to do things